<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NitpickClass</class>
 <widget class="QMainWindow" name="NitpickClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>870</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Nitpick</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>750</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>140</y>
      <width>630</width>
      <height>580</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>Create</string>
     </attribute>
     <widget class="QPushButton" name="createTestsButton">
      <property name="geometry">
       <rect>
        <x>195</x>
        <y>60</y>
        <width>220</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Create Tests</string>
      </property>
     </widget>
     <widget class="QPushButton" name="createMDFileButton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>180</y>
        <width>220</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Create MD file</string>
      </property>
     </widget>
     <widget class="QPushButton" name="createAllMDFilesButton">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>180</y>
        <width>220</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Create all  MD files</string>
      </property>
     </widget>
     <widget class="QPushButton" name="createTestsOutlineButton">
      <property name="geometry">
       <rect>
        <x>195</x>
        <y>120</y>
        <width>220</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Create Tests Outline</string>
      </property>
     </widget>
     <widget class="QPushButton" name="createRecursiveScriptButton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>300</y>
        <width>220</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Create Recursive Script</string>
      </property>
     </widget>
     <widget class="QPushButton" name="createAllRecursiveScriptsButton">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>300</y>
        <width>220</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Create all  Recursive Scripts</string>
      </property>
     </widget>
     <widget class="QPushButton" name="createTestAutoScriptButton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>240</y>
        <width>220</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Create testAuto script</string>
      </property>
     </widget>
     <widget class="QPushButton" name="createAllTestAutoScriptsButton">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>240</y>
        <width>220</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Create all testAuto scripts</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Windows</string>
     </attribute>
     <widget class="QPushButton" name="hideTaskbarButton">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>130</y>
        <width>211</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Hide Windows Taskbar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="showTaskbarButton">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>200</y>
        <width>211</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Show Windows Taskbar</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Run</string>
     </attribute>
     <widget class="QPushButton" name="runNowButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Run now</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="daysGroupBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>91</width>
        <height>241</height>
       </rect>
      </property>
      <property name="title">
       <string>Days</string>
      </property>
      <widget class="QCheckBox" name="sundayCheckBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>80</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Sunday</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="wednesdayCheckBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>80</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Wednesday</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="tuesdayCheckBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>80</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Tuesday</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="thursdayCheckBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>80</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Thursday</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="fridayCheckBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>80</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Friday</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="saturdayCheckBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>80</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Saturday</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="mondayCheckBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>80</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Monday</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="timesGroupBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>240</y>
        <width>161</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string>Times</string>
      </property>
      <widget class="QTimeEdit" name="timeEdit1">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>118</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QTimeEdit" name="timeEdit2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>118</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QTimeEdit" name="timeEdit3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>118</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QTimeEdit" name="timeEdit4">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>140</y>
         <width>118</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QCheckBox" name="timeEdit1checkBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>23</y>
         <width>21</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QCheckBox" name="timeEdit2checkBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>63</y>
         <width>21</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QCheckBox" name="timeEdit3checkBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>103</y>
         <width>21</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QCheckBox" name="timeEdit4checkBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>143</y>
         <width>21</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="setWorkingFolderButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>161</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Set Working Folder</string>
      </property>
     </widget>
     <widget class="QLabel" name="workingFolderLabel">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>20</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>(not set...)</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>210</y>
        <width>311</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="workingFolderLabel_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>170</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Status:</string>
      </property>
     </widget>
     <widget class="QLabel" name="statusLabel">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>170</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>#######</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxServerless">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>120</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If unchecked, will not show results during evaluation&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Server-less</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxRunLatest">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>120</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If unchecked, will not show results during evaluation&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Run Latest</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="workingFolderLabel_3">
      <property name="geometry">
       <rect>
        <x>128</x>
        <y>95</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>URL</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="urlLineEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>100</y>
        <width>451</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Evaluate</string>
     </attribute>
     <widget class="QCheckBox" name="checkBoxInteractiveMode">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>180</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If unchecked, will not show results during evaluation&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Interactive Mode</string>
      </property>
     </widget>
     <widget class="QPushButton" name="evaluateTestsButton">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>170</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Evaluate Test</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Web Interface</string>
     </attribute>
     <widget class="QPushButton" name="updateTestRailRunResultsButton">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>220</y>
        <width>160</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Update Run Results</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="createPythonScriptRadioButton">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>100</y>
        <width>95</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Python</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="createTestRailRunButton">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>160</y>
        <width>160</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Create Run</string>
      </property>
     </widget>
     <widget class="QPushButton" name="createTestRailTestCasesButton">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>100</y>
        <width>160</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Create Test Cases</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="createXMLScriptRadioButton">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>120</y>
        <width>95</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>XML</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>601</width>
        <height>300</height>
       </rect>
      </property>
      <property name="title">
       <string>TestRail</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>350</y>
        <width>601</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string>Amazon Web Services</string>
      </property>
      <widget class="QPushButton" name="createWebPagePushButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>30</y>
         <width>160</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Create Web Page</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="updateAWSCheckBox">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>42</y>
         <width>111</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Update AWS</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="awsURLLineEdit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>561</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
     <zorder>groupBox</zorder>
     <zorder>updateTestRailRunResultsButton</zorder>
     <zorder>createPythonScriptRadioButton</zorder>
     <zorder>createTestRailRunButton</zorder>
     <zorder>createTestRailTestCasesButton</zorder>
     <zorder>createXMLScriptRadioButton</zorder>
     <zorder>groupBox_2</zorder>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>110</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>GitHub Branch</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>110</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>GitHub User</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>760</y>
      <width>255</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="userLineEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="branchLineEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionContent"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionContent">
   <property name="text">
    <string>Content</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>userLineEdit</tabstop>
  <tabstop>branchLineEdit</tabstop>
  <tabstop>createTestsButton</tabstop>
  <tabstop>createMDFileButton</tabstop>
  <tabstop>createAllMDFilesButton</tabstop>
  <tabstop>createTestsOutlineButton</tabstop>
  <tabstop>createRecursiveScriptButton</tabstop>
  <tabstop>createAllRecursiveScriptsButton</tabstop>
  <tabstop>createTestAutoScriptButton</tabstop>
  <tabstop>createAllTestAutoScriptsButton</tabstop>
  <tabstop>hideTaskbarButton</tabstop>
  <tabstop>showTaskbarButton</tabstop>
  <tabstop>runNowButton</tabstop>
  <tabstop>sundayCheckBox</tabstop>
  <tabstop>wednesdayCheckBox</tabstop>
  <tabstop>tuesdayCheckBox</tabstop>
  <tabstop>thursdayCheckBox</tabstop>
  <tabstop>fridayCheckBox</tabstop>
  <tabstop>saturdayCheckBox</tabstop>
  <tabstop>mondayCheckBox</tabstop>
  <tabstop>timeEdit1</tabstop>
  <tabstop>timeEdit2</tabstop>
  <tabstop>timeEdit3</tabstop>
  <tabstop>timeEdit4</tabstop>
  <tabstop>timeEdit1checkBox</tabstop>
  <tabstop>timeEdit2checkBox</tabstop>
  <tabstop>timeEdit3checkBox</tabstop>
  <tabstop>timeEdit4checkBox</tabstop>
  <tabstop>setWorkingFolderButton</tabstop>
  <tabstop>plainTextEdit</tabstop>
  <tabstop>checkBoxServerless</tabstop>
  <tabstop>checkBoxRunLatest</tabstop>
  <tabstop>urlLineEdit</tabstop>
  <tabstop>checkBoxInteractiveMode</tabstop>
  <tabstop>evaluateTestsButton</tabstop>
  <tabstop>updateTestRailRunResultsButton</tabstop>
  <tabstop>createPythonScriptRadioButton</tabstop>
  <tabstop>createTestRailRunButton</tabstop>
  <tabstop>createTestRailTestCasesButton</tabstop>
  <tabstop>createXMLScriptRadioButton</tabstop>
  <tabstop>createWebPagePushButton</tabstop>
  <tabstop>updateAWSCheckBox</tabstop>
  <tabstop>awsURLLineEdit</tabstop>
  <tabstop>closeButton</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
